src/lib.rs

```diff
@@ -69,8 +66,6 @@ impl Plugin for AppPlugin {
 /// call above.
 #[derive(SystemSet, Debug, Clone, Copy, Eq, PartialEq, Hash)]
 enum AppSet {
-    /// Tick timers.
-    TickTimers,
```

Comment:
Why remove this?

Reply:
splash timer runs before everything else, but animation timer runs after inputs
so this system set would only be used by a single system, i settled for chain

Reply:
I'd argue that update animation should be split into two systems, one to tick the timer and one to update the animation, but this is fine I guess.

Reply:
I agree with @benfrankel here\

Reply:
Done